src/main/java/TaskList.java

```diff
@@ -0,0 +1,119 @@
+package main.java;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * The TaskList contains the list of tasks and has operations to add/delete/etc. tasks in the list
+ *
+ * @author Lio
+ */
+public class TaskList {
+    private List<Task> list = new ArrayList<>();
+
+    /**
+     * Constructor of empty list
+     */
+    public TaskList() {
+    }
```

Comment:
Perhaps could remove this if it is not necessary, since there is a default constructor?